packages/ui/Marquee/MarqueeDecorator.js

```diff
@@ -285,8 +285,7 @@ const MarqueeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 		}
 
 		static contextTypes = {
-			...contextTypes,
-			...stateContextTypes
+			...contextTypes
```

Comment:
No need to clone the object here. Pass by ref is okay now that it's only 1 `contextTypes`.